test/e2e/upgrade/upgrade.go

```diff
@@ -111,35 +111,44 @@ func SetUpgradeAbortAt(policy string) error {
 	return fmt.Errorf("abort-at must be empty, set to 'random', or an integer in [0,100], inclusive")
 }
 
-var _ = g.Describe("[sig-arch][Feature:ClusterUpgrade]", func() {
+var _ = g.Describe("[sig-arch][Feature:ClusterUpgrade] Upgrading", func() {
 	f := framework.NewDefaultFramework("cluster-upgrade")
 	f.SkipNamespaceCreation = true
 	f.SkipPrivilegedPSPBinding = true
-
-	g.It("Cluster should remain functional during upgrade [Disruptive]", func() {
-		config, err := framework.LoadConfig()
+	var (
+		config *rest.Config
+		client *configv1client.Clientset
+		dynamicClient dynamic.Interface
+		upgCtx *upgrades.UpgradeContext
+	)
+
+	g.It("should load the test configuration", func() {
```

Comment:
You can't do this, it gets processed in the upgrade step.

Dynamic tests cannot be done based on cluster info.